docs/sources/enterprise/request-security.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Request security"
+description = "Grafana Enterprise request security"
+keywords = ["grafana", "security", "enterprise"]
+weight = 110
++++
+
+# Request security
+
+> **Note:** Available in Grafana Enterprise v7.4+.
+
+Request security introduces ways to limit requests from the Grafana server and focuses on requests generated by users.
```

Comment:
```suggestion
Request security introduces ways to limit requests from the Grafana server and focuses on requests generated by users.
```

What does "focuses" mean here? "Request security focuses on ..."

Reply:
I'm trying to signal that the focus is requests that are a direct result of user input. Datasource config, alerting. This feature doesn't cover all requests, like the request to the database or the ldap system. Those are done when setting up grafana and are not covered. I'm having a hard time finding the right way to make this distinction.

Reply:
Thank you for the added context. How do you like this alternative?

Reply:
My gut says that a human can focus, or make a machine focus, rather than a machine focusing (read concentrating) on something.

Reply:
Makes sense to me, thanks!